VAMobile/src/screens/HealthScreen/SecureMessaging/StartNewMessage/StartNewMessage.tsx

```diff
@@ -357,7 +317,14 @@ function StartNewMessage({ navigation, route }: StartNewMessageProps) {
             })
           },
           onError: () => {
-            showSnackBar(snackbarMessages.errorMsg, dispatch, () => saveDraft(params, mutateOptions), false, true)
+            showSnackBar(
+              snackbarMessages.errorMsg,
+              dispatch,
+              // passing messageDataRef to ensure we have the latest messageData
+              () => saveDraft({ messageData: messageDataRef.current }, mutateOptions),
```

Comment:
Why does this one get away with skipping the category assignment?

Reply:
The category is included in messageDataRef.current